lib/task/migrate/QubitMigrate.class.php

```diff
@@ -930,16 +930,20 @@ public static function updateIndexes($indexes)
 
   public static function updateForeignKeys($foreignKeys)
   {
+    $dbname = QubitPdo::fetchColumn('select database();');
```

Comment:
Maybe we can get it from the config to avoid the query on each foreign key update, not a big deal though.

Reply:
@jraddaoui I looked for a way to get the DB name from the config, but I think it requires parsing the DSN string :(